corehq/apps/app_manager/templates/app_manager/partials/define_case_type_modal.html

```diff
@@ -0,0 +1,78 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<!-- B5 TODO: Rewrite this modal to use B5 built-in utility -->
```

Comment:
All modals get tagged by the B5 migration. Is there anything in particular about this modal that means it needs attention beyond the usual B3 => B5 modal changes?

Reply:
(Is it about the new styles in `new_case_type_modal.less`?)

Reply:
Yes, it is about the new styles! Will a comment in `new_case_type_modal.less` make more sense?

Reply:
Yeah, in the styles file makes sense to me. Whoever migrates app manager will run into that file, but without a comment they might just mechanically translate the LESS to SaSS. So a note in that file saying to use B5 utilities sounds good.

Reply:
Removed the less file in 199c26a20f9929ad78a74aba70044562357f29fe so also removed the B5 migration comment 😬